File: PB - More Exercises Python/2.0 Conditional Statements - More Exercises/05. Firm.py
from math import trunc

hours, day, workers = [int(input()) for _ in range(3)]

remnant_time = trunc(day * 0.90 * 8)
extra_work_time = workers * (2 * day)
total_time = remnant_time + extra_work_time

if total_time >= hours:
    print(f'Yes!{total_time - hours} hours left.')
else:
    print(f'Not enough time!{hours - total_time} hours needed.')


'''
90
7
3
'''

File: PB - More Exercises GO/1.0 Simple Operations and Calculations - More Exercises/2.0 Conditional Statements - More Exercises/05. Firm.go
package main

import (
	"fmt"
	"math"
)

func main() {
	var hours, day, workers float64
	fmt.Scan(&hours, &day, &workers)

	remnantTime := math.Trunc(day * 0.90 * 8)
	extraWorkTime := workers * (2 * day)
	totalTime := remnantTime + extraWorkTime

	if totalTime >= hours {
		fmt.Printf("Yes!%.0f hours left.", totalTime-hours)
	} else {
		fmt.Printf("Not enough time!%.0f hours needed.", hours-totalTime)
	}
}

/*
name   :05. Firm
input  :90 7 3
output :Yes!2 hours left.
*/
